DbcParserLib/Model/MultiplexingRange.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DbcParserLib.Model
+{
+    public class MultiplexingRange
+    {
+        public bool IsRange { get; }
```

Comment:
What about expanding the valid values?
So that `1-1, 5-7` is exploded into `[1, 5, 6, 7]`.
I was wondering if that would simplify, after all range or not, from a decoding point of view, we only need to read a Muxer value and pick the right layout based on that value.

Just reasoning